.github/workflows/unit-test-rust.yaml

```
@@ -0,0 +1,51 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+name: "Run unit tests (Rust)"
+on:
+  push:
+    branches:
+      - main
+      - "release/v*.*"
+      - "!release/v*.*.*"
+    paths:
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 23: did not find expected '-' indicator

(syntax-check)